mainnet/draft_genesis.json

```diff
@@ -0,0 +1,682 @@
+              "revision_height": "0"
+            },
+            "timestamp": "600000000000"
+          }
+        }
+      }
+    },
+    "interchainaccounts": {
+      "controller_genesis_state": {
+        "active_channels": [],
+        "interchain_accounts": [],
+        "ports": [],
+        "params": {
+          "controller_enabled": true
+        }
+      },
+      "host_genesis_state": {
+        "active_channels": [],
+        "interchain_accounts": [],
+        "port": "icahost",
+        "params": {
+          "host_enabled": true,
+          "allow_messages": [
+            "*"
+          ]
+        }
+      }
+    },
+    "leveragelp": {
+      "params": {
+        "leverage_max": "10.000000000000000000",
+        "max_open_positions": "9999999999999",
+        "pool_open_threshold": "0.200000000000000000",
+        "safety_factor": "1.100000000000000000",
+        "whitelisting_enabled": false,
+        "epoch_length": "9999999999999",
+        "fallback_enabled": false,
+        "number_per_block": "100"
+      },
+      "pool_list": [],
+      "position_list": [],
+      "address_whitelist": []
+    },
+    "masterchef": {
+      "params": {
+        "lp_incentives": null,
+        "reward_portion_for_lps": "0.600000000000000000",
+        "reward_portion_for_stakers": "0.250000000000000000",
+        "max_eden_reward_apr_lps": "0.500000000000000000",
+        "supported_reward_denoms": [],
+        "protocol_revenue_address": "NEED THIS ADRESS"
+      },
+      "external_incentives": [],
+      "external_incentive_index": "0",
+      "pool_infos": [],
+      "pool_reward_infos": [],
+      "user_reward_infos": [],
+      "pool_rewards_accum": []
+    },
+    "oracle": {
+      "params": {
+        "band_channel_source": "channel-1",
+        "oracle_script_id": "37",
+        "multiplier": "18",
+        "ask_count": "4",
+        "min_count": "3",
+        "fee_limit": [
+          {
+            "denom": "uband",
+            "amount": "30"
+          }
+        ],
+        "prepare_gas": "600000",
+        "execute_gas": "600000",
+        "client_id": "band_price_id",
+        "band_epoch": "band_epoch",
+        "price_expiry_time": "86400",
+        "life_time_in_blocks": "1"
+      },
+      "port_id": "oracle",
+      "asset_infos": [
+        {
+          "denom": "uelys",
+          "display": "ELYS",
+          "band_ticker": "ELYS",
+          "elys_ticker": "ELYS",
+          "decimal": "6"
+        },
+        {
+          "denom": "ueden",
+          "display": "EDEN",
+          "band_ticker": "EDEN",
+          "elys_ticker": "EDEN",
+          "decimal": "6"
+        },
+        {
+          "denom": "uedenb",
+          "display": "EDENB",
+          "band_ticker": "EDENB",
+          "elys_ticker": "EDENB",
+          "decimal": "6"
+        }
+      ],
+      "prices": [],
+      "price_feeders": [
+        {
+          "feeder": "elys10d07y265gmmuvt4z0w9aw880jnsr700j6z2zm3",
```

Comment:
@avkr003 that one seems to be used in testnet as well is it intentional?

Reply:
gov address